### PR TITLE
GEODE-5971: refactor more to use ResultModel

### PR DESCRIPTION
**Pull request by jinmeiliao**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jinmeiliao** [comment]
@Petahhh

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java [hunk 1]
```diff
@@ -50,61 +48,52 @@
...
+      for (String regionName : regionNames) {
+        resultData.accumulate("List of regions", regionName);
       }
-    } catch (FunctionInvocationTargetException e) {
-      result = ResultBuilder.createGemFireErrorResult(CliStrings
-          .format(CliStrings.COULD_NOT_EXECUTE_COMMAND_TRY_AGAIN, CliStrings.LIST_REGION));
-    } catch (Exception e) {
-      result = ResultBuilder
-          .createGemFireErrorResult(CliStrings.LIST_REGION__MSG__ERROR + " : " + e.getMessage());
     }
```
Comment (onichols-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java]: why do we no longer need exception handling here?

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/NetstatCommand.java [hunk 2]
```diff
@@ -156,34 +152,15 @@ public Result netstat(
...
-      result = ResultBuilder.createUserErrorResult(e.getMessage());
-    } catch (RuntimeException e) {
-      LogWrapper.getInstance(getCache())
-          .info(CliStrings.format(
-              CliStrings.NETSTAT__MSG__ERROR_OCCURRED_WHILE_EXECUTING_NETSTAT_ON_0,
-              new Object[] {Arrays.toString(members)}), e);
-      result = ResultBuilder.createGemFireErrorResult(
-          CliStrings.format(CliStrings.NETSTAT__MSG__ERROR_OCCURRED_WHILE_EXECUTING_NETSTAT_ON_0,
-              new Object[] {Arrays.toString(members)}));
     } finally {
```
Comment (onichols-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/NetstatCommand.java]: why do we no longer need exception handling here?

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java (hunk 1 above)
Comment (jinmeiliao) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java]: the CommandExecutor will catch all exceptions and do these things.

Comment (onichols-pivotal) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java]: cool!

**Pull request merged**